plugin/manager.go

```diff
@@ -289,13 +290,46 @@ func attachToLog(id string) func(libcontainerd.IOPipe) error {
 }
 
 func validatePrivileges(requiredPrivileges, privileges types.PluginPrivileges) error {
-	// todo: make a better function that doesn't check order
-	if !reflect.DeepEqual(privileges, requiredPrivileges) {
+	if !isEqual(requiredPrivileges, privileges, isEqualPrivilege) {
 		return errors.New("incorrect privileges")
 	}
+
 	return nil
 }
 
+func isEqual(arrOne, arrOther types.PluginPrivileges, compare func(x, y types.PluginPrivilege) bool) bool {
```

Comment:
hmmm... sorry to go back and forth here (and maybe hold off on updating for someone else to comment)... but I think the whole data structure should match. Ordering probably isn't important (since this could be different depending on serialization and such), but I would expect descriptions to be the same even if functionally the description does nothing. It's more "Here's what the user is agreeing to", which is based on description.